scuttlebutt/src/lib.rs

```diff
@@ -39,15 +39,53 @@ use tracing::{debug, error};
 use crate::digest::Digest;
 use crate::message::ScuttleButtMessage;
 use crate::serialize::Serializable;
-pub use crate::state::ClusterState;
+pub use crate::state::{ClusterState, SerializableClusterState};
 use crate::state::NodeState;
 
 /// Map key for the heartbeat node value.
 pub(crate) const HEARTBEAT_KEY: &str = "heartbeat";
 
 pub type Version = u64;
 
-pub type NodeId = String;
+// pub type NodeId = String;
+
+#[derive(Default, Clone, Hash, Eq, PartialEq, PartialOrd, Ord, Debug, Serialize, Deserialize)]
```

Comment:
We need doc here. Copy pasting your doc is ok.

Reply:
Adding doc, also part of the doc will be in Quickwit where QuickwitNodeId is defined.